app/src/main/java/com/edu/uz/currency/currencyapp/retrofit/NbpClient.java

```diff
@@ -0,0 +1,45 @@
+package com.edu.uz.currency.currencyapp.retrofit;
```

Comment:
Ogólnie we wszystkich projektach jakie miałem bylo to wrzucane do paczki "rest" albo cos w tym stylu, może taka?